build.sbt

```diff
@@ -305,10 +309,10 @@ lazy val examples = crossProject(JVMPlatform, JSPlatform)
   .in(file("examples"))
   .settings(stdSettings("examples"))
   .settings(crossProjectSettings)
-  .settings(macroSettings)
+  .settings(macroExpansionSettings)
   .settings(testFrameworks += new TestFramework("zio.test.sbt.ZTestFramework"))
   .jsSettings(libraryDependencies += "io.github.cquiroz" %%% "scala-java-time" % "2.0.0-RC5" % Test)
-  .dependsOn(testRunner)
+  .dependsOn(macros, testRunner)
```

Comment:
Do we need this? I would like to avoid making ZIO Test depend on ZIO Macros and instead have it be an optional feature that users can include if they want? Also it doesn't seem to be a dependency on other platforms right now which seems to indicate it is not necessary.

Reply:
It's not ZIO Test - it's examples. I agree with you on macros being optional.

Reply:
Ah sorry I misread.